tests/Jira/Issues/WalkerTest.php

```diff
@@ -265,6 +265,11 @@ public function testCounting()
 		);
 	}
 
+	public function testYouCanSpecifyFieldsetsToExpandWithASearch()
```

Comment:
What do I have to do to enable this test to function?

Reply:
What function? The test is empty.

Reply:
er - precisely ;-)  need to know how to fill in test function.

Reply:
You can look at other tests in test suite for examples.